### PR TITLE
Added Speedtest

### PR DESCRIPTION
**Pull request by angrycuban13**
# Pull request

**Purpose**
Describe the problem or feature in addition to a link to the issues.

**Approach**
How does this change address the problem?

**Open Questions and Pre-Merge TODOs**
Check all boxes as they are completed

- [ ] Use github checklists. When solved, check the box and explain the answer.

**Learning**
Describe the research stage
Links to blog posts, patterns, libraries or addons used to solve this problem

**Requirements**
Check all boxes as they are completed

- [ ] These changes meet the standards for [contributing](https://github.com/GhostWriters/DockSTARTer/blob/master/.github/CONTRIBUTING.md).
- [ ] I have read the [code of conduct](https://github.com/GhostWriters/DockSTARTer/blob/master/.github/CODE_OF_CONDUCT.md).

### CODE REVIEW

File: compose/.apps/speedtest/speedtest.labels.yml [hunk 1]
```diff
@@ -0,0 +1,11 @@
...
+  speedtest:
+    labels:
+      com.dockstarter.appinfo.description: "This program runs a speedtest check every hour and graphs the results."
+      com.dockstarter.appinfo.deprecated: "false"
+      com.dockstarter.appinfo.nicename: "speedtest"
+      com.dockstarter.appvars.speedtest_enabled: "false"
+      com.dockstarter.appvars.speedtest_network_mode: ""
+      com.dockstarter.appvars.speedtest_port_80: "80"
+      com.dockstarter.appvars.speedtest_base_path: "/"
+      com.dockstarter.appvars.speedtest_ookla_eula: "true"
```
Comment (nemchik) [inline compose/.apps/speedtest/speedtest.labels.yml:11]:
```suggestion
      com.dockstarter.appvars.speedtest_base_path: "/"
      com.dockstarter.appvars.speedtest_enabled: "false"
      com.dockstarter.appvars.speedtest_network_mode: ""
      com.dockstarter.appvars.speedtest_ookla_eula: "true"
      com.dockstarter.appvars.speedtest_port_80: "80"
```

File: compose/.apps/speedtest/speedtest.yml [hunk 2]
```diff
@@ -0,0 +1,18 @@
...
+      - TZ=${TZ}
+    logging:
+      driver: json-file
+      options:
+        max-file: ${DOCKERLOGGING_MAXFILE}
+        max-size: ${DOCKERLOGGING_MAXSIZE}
+    restart: unless-stopped
+    volumes:
+      - /etc/localtime:/etc/localtime:ro
+      - ${DOCKERCONFDIR}/speedtest:/config
```
Comment (nemchik) [inline compose/.apps/speedtest/speedtest.yml:18]: Missing shared dir

Comment (angrycuban13) [reply compose/.apps/speedtest/speedtest.yml:18]: I didn't think this container would need it specifically lol, I'll fix it.

File: compose/.apps/speedtest/speedtest.labels.yml (hunk 1 above)
Comment (angrycuban13) [reply compose/.apps/speedtest/speedtest.labels.yml:11]: I thought I had gotten it right this time :(

File: compose/.apps/speedtest/speedtest.yml (hunk 2 above)
Comment (nemchik) [reply compose/.apps/speedtest/speedtest.yml:18]: As a standard across all containers we always include it.

**Pull request merged**